setup.py

```diff
@@ -33,7 +33,7 @@
     description = 'Pygments is a syntax highlighting package written in Python.',
     long_description = __doc__,
     keywords = 'syntax highlighting',
-    packages = find_packages(),
+    packages = find_packages(exclude=['tests']),
```

Comment:
I'd prefer `include=['pygments']`, since it prevents further accidents.

Reply:
Done!